package.json

```
@@ -9,29 +9,31 @@
   "license": "Apache-2.0",
   "description": "IBM's Watson Health Design Pattern & Assets Library Exported Tests architecture",
   "engines": {
-    "node": "^10"
+    "node": "10"
```

```suggestion
    "node": "^10"
```

They can use node v10 and up